src/drive/web/modules/views/OnlyOffice/Toolbar/index.jsx

```diff
@@ -16,7 +16,9 @@ import Sharing from 'drive/web/modules/views/OnlyOffice/Toolbar/Sharing'
 
 const Toolbar = () => {
   const { isMobile } = useBreakpoints()
-  const { fileId, isPublic, isReadOnly } = useContext(OnlyOfficeContext)
+  const { fileId, isPublic, isReadOnly, isEditorReady } = useContext(
```

Comment:
`isPublic` refers to the sharing, right? 

`isReadOnly` refers to the editor or to the sharing too?

`isEditorReady` refers to the editor ?

Reply:
> isPublic refers to the sharing, right?
Oui et non, en réalité le isPublic est forcé à true quand on vient d'une route publique, il n'y a aucune logique derrière

> isReadOnly refers to the editor or to the sharing too?
Lui n'est setté qu'après retour de la stack, ça vient de la config qu'on récupère

> isEditorReady refers to the editor ?
Ca c'est côté front une fois que l'éditor OO est bien opérationnel et qu'on peut modifier/consulter le doc. Donc oui ça fait référence à l'éditeur.

Reply:
OK so isReadOnly refers to the OO config. So it should be isEditorReadOnly? To be consistent with `isEditorReady`?